msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -64,6 +70,20 @@
     <PackageReference Include="System.Xml.XDocument" Version="4.3.0" />
     <PackageReference Include="System.Security.Cryptography.X509Certificates" Version="4.3.0" Condition="'$(TargetFramework)' != 'netstandard1.1'" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
+    <!-- Remove UI specific logic as it is not supported yet -->
```

Comment:
This is where most of the removal logic sits - I excluded a bunch of public APIs